src/common/UploadOptions.svelte

```diff
@@ -1,64 +1,118 @@
 <script>
-  import Omniselect from "./Omniselect";
-  import Checkbox from "./Checkbox";
   import {
     defaultLanguage,
     languages,
     textractLanguages,
-  } from "@/api/languages";
-  import { orgsAndUsers, initOrgsAndUsers } from "@/manager/orgsAndUsers";
+  } from "@/api/languages.js";
+  import { orgsAndUsers, initOrgsAndUsers } from "@/manager/orgsAndUsers.js";
   import { onMount } from "svelte";
   import { _ } from "svelte-i18n";
 
   export let language = defaultLanguage;
   export let forceOcr = false;
   export let ocrEngine = "tess4";
 
-  const showOcrEngine = $orgsAndUsers.me.feature_level > 0;
+  const hasTextract = $orgsAndUsers.me.feature_level > 0;
 
-  $: selectLanguages = ocrEngine == "textract" ? textractLanguages : languages;
+  // is this brittle if there isn't a default language?
```

Comment:
I had the default language populated at one point, but I took it out. The way this works now, using `<datalist>` and a text input, it will autocomplete based on whatever is in the text field. If English is already there, it looks like your only language choices are `English` and `Middle English`, unless you clear it. This way, at least, you can see all language choices. I'm open to ideas on how to make this a smoother experience.